src/main/java/it/pagopa/interop/probing/scheduler/dto/impl/EserviceContent.java

```diff
@@ -11,7 +12,7 @@
 
 @Data
 @Builder
-public class EserviceContent {
+public class EserviceContent implements Dto {
```

Comment:
Named class with postfix Dto

Reply:
Actually this is not a dto, it is an internal data structure of a Response object, should i leave it without interface?

Reply:
Then it should have a Response interface